### PR TITLE
Magento 2.2 Compatibility Patch

### PR DESCRIPTION
**Pull request by Silarn**
There was one breaking change in Magento 2.2 - the Mail Transport interface added a new getMessage function requirement.

This change should be backward compatible.

### CODE REVIEW

**Comment by centerax** [comment]
thanks @Silarn . Once the failing test passes, we will merge it.

File: Model/Transport.php [hunk 1]
```diff
@@ -82,6 +82,10 @@ public function sendMessage()
         return true;
     }
 
+    public function getMessage() {
```
Comment (centerax) [inline Model/Transport.php:85]: please put the opening brace on a new line.

**Review by centerax — CHANGES_REQUESTED** [review]

**Comment by Silarn** [comment]
Sorry I didn't get to this this weekend, but it looks like it's been handled.

**Comment by centerax** [comment]
its sorted, thanks @Silarn for your contribution.

**Pull request closed without merging**